src/ansible_navigator/steps.py

```diff
@@ -122,6 +128,90 @@ def _value_check(value: Any, want: Union[type, Tuple[type, ...]]) -> None:
             raise ValueError(f"wanted {want}, got {type(value)}")
 
 
+TValueType = TypeVar("TValueType")
```

Comment:
Prefer just `T`, type variables are usually (by convention) one uppercase letter.

Reply:
Updated to just T. I don't love that Pylance highlights it as a constant, but that sounds like a Pylance problem.